src/Configuration/UserDefinition.php

```diff
@@ -34,7 +34,7 @@ private function buildRootDefinition(TreeBuilder $treeBuilder): ArrayNodeDefinit
                     ->scalarPrototype()
                     ->validate()
                         ->ifTrue(function ($value) {
-                            return preg_match('~' . SchemaDefinition::REGEX_SCHEMA_NAME . '~', $value) !== 1;
+                            return !SchemaDefinition::isSchemaNameValid($value);
                         })
                         ->thenInvalid('Schema name can only contain alphanumeric characters and underscore')
```

Comment:
není to součástí PR, ale `underscore` => `underscores`

Reply:
Upravím v navazujícím.